Sources/PackageModel/Manifest/Manifest.swift

```diff
@@ -106,16 +105,16 @@ public final class Manifest: Sendable {
         path: AbsolutePath,
         packageKind: PackageReference.Kind,
         packageLocation: String,
-        defaultLocalization: String? = nil,
+        defaultLocalization: String?,
         platforms: [PlatformDescription],
-        version: TSCUtility.Version? = nil,
-        revision: String? = nil,
+        version: TSCUtility.Version?,
+        revision: String?,
```

Comment:
Are version and revision mutually exclusive?

Reply:
not that I know. Am I missing something?